app/models/user.rb

```diff
@@ -76,6 +75,11 @@ def to_s
     twitch_display_name || name || 'somebody'
   end
 
+  def rivals
+    User.joins('INNER JOIN rivalries ON rivalries.to_user_id = users.id AND rivalries.from_user_id = ' + id.to_s)
```

Comment:
Instead of string concatenation, use SQL interpolation like
```ruby
User.joins('INNER JOIN ... AND rivalries.from_user_id = ?', id)
```
which will sanitize inputs to prevent SQL injection. It's just IDs in this case, but it's still a good practice (and who knows whether there might be a separate attack in the future to inject or set custom IDs).